packages/rich-text/src/to-html-string.js

```diff
@@ -19,14 +19,16 @@ import { toTree } from './to-tree';
 /**
  * Create an HTML string from a Rich Text value.
  *
- * @param {Object}        $1       Named argements.
- * @param {RichTextValue} $1.value Rich text value.
+ * @param {Object}        $1                      Named argements.
+ * @param {RichTextValue} $1.value                Rich text value.
+ * @param {?boolean}      [$1.preserveWhiteSpace] Preserves newlines if true.
```

Comment:
```suggestion
 * @param {boolean}      [$1.preserveWhiteSpace] Preserves newlines if true.
```

I think `[]` indicates that the parameter is optional, so I think `?` is unnecessary.

Or we may want to specify a default value.

```javascript
 * @param {boolean}      [$1.preserveWhiteSpace=false] Preserves newlines if true.
```